scripts/pipelines/github/templates/build/build-pipeline.yml.template

```
@@ -0,0 +1,33 @@
+name: $pipelineName
+
+on:
+  workflow_dispatch:
+    inputs:
+      target:
+        required: false
+        default: 'target'
+
+# mark to insert additional artifact input #
+
+env:
+  targetPath: ${{ github.event_name == 'push' && format('target') || github.event.inputs.target }}
```

As seen with Albert last time, we can defin the env var in a  simpler way
```suggestion
  targetPath: ${{ github.event.inputs.target  || 'target' }}
```